Userland/Applications/Browser/Tab.cpp

```diff
@@ -995,11 +995,14 @@ void Tab::show_storage_inspector()
         storage_window->resize(500, 300);
         storage_window->set_title("Storage Inspector");
         storage_window->set_icon(g_icon_bag.cookie);
-        m_storage_widget = storage_window->set_main_widget<StorageWidget>();
-        m_storage_widget->on_update_cookie = [this](Web::Cookie::Cookie cookie) {
-            if (view().on_update_cookie)
-                view().on_update_cookie(move(cookie));
-        };
+        if (auto storage_widget = StorageWidget::create(); !storage_widget.is_error()) {
```

Comment:
Are we sure that this window should just be empty if the widget fails to initialize?